docusaurus.config.js

```diff
@@ -108,10 +108,6 @@ const navbarItems = [
         label: 'Trademark Guidelines',
         to: 'about/trademark',
       },
-      {
-        label: 'Get the Book',
-        to: 'about/book',
-      },
```

Comment:
If we're using this page as the central place to stay up to date about Book progress, I'd probably keep the link in the menu. As having to know hidden URL exists isn't a great way to find the page again :sweat_smile:

Reply:
Let me update the label for it then to "Tauri Book", "Get the Book" makes it sound like it's available